src/components/bookForm.css

```diff
@@ -0,0 +1,77 @@
+.book-input h2 {
+  text-align: center;
+  font-weight: 800;
+}
+
+.book-input form {
+  width: auto;
+  height: auto;
+  padding: 11px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  align-items: center;
+}
+
+.book-input input {
+  width: 80%;
+  padding: 11px;
+  border-radius: 5px;
+}
+
+.books-option {
+  padding: 11px;
+  font-size: 16px;
+  border-radius: 6px;
+  width: 87%;
+}
+
+.book-input button {
+  font-size: 16px;
+  font-weight: bold;
+  padding: 11px;
+  border-radius: 6px;
+  width: 30%;
+}
+
```

Comment:
- [ ] Kindly use the same colors and fonts as the Zeplin design for the button and the whole form.